tools.html

```diff
@@ -2,7 +2,7 @@
 layout: general
 highlight_section: tools
 title: Get tools
-description: Prepare your environment with DevAssistant. Collaborate with Docker and Vagrant.
+description: Prepare your environment with DevAssistant. Collaborate with Docker, Vagrant and Virt-builder.
```

Comment:
Yeah -- agreed on not mentioning all the tools. But, currently I don't think the sentence isn't too cumbersome to read. That said, feel free to modify it as you see fit.